packages/opentelemetry-metrics/src/ObserverResult.ts

```diff
@@ -26,6 +26,8 @@ export class ObserverResult implements TypeObserverResult {
   values: Map<Labels, number> = new Map<Labels, number>();
 
   observe(value: number, labels: Labels): void {
-    this.values.set(labels, value);
+    if (value >= 0) {
```

Comment:
Is an observer required to be greater than 0?

Reply:
Good catch, I have found the bug, the monotonic previously was implemented as something that needs to be greater than 0 which is not true. Refactoring the way the monotonic compares value to previous one, will push fix soon